### ISSUE TITLE
It does not support android 4.0.3

### ISSUE THREAD

**Issue by manoj529** [danielzen/todo-ng-pouchdb#1]
I have created a sample ionic app and replaced the www app. the apk file thus generated works fine android 4.1 or later but when i tried to install it in android 4.0.3 it throws error like below stated

"There is a problem parsing the package"

**Comment by lufvape** [comment]
did you prove this todo app in android 4.1? Is it working fine; seems Android 5 is not

**Comment by manoj529** [comment]
Yeah i tried on android 4.1 and worked fine @lfvargas1746  . I did not tested on android 5.0
